compiler/rustc_ast_passes/src/ast_validation.rs

```diff
@@ -917,16 +917,16 @@ impl<'a> Visitor<'a> for AstValidator<'a> {
                 walk_list!(self, visit_attribute, &item.attrs);
                 return; // Avoid visiting again.
             }
-            ItemKind::Fn(box Fn { defaultness, sig, generics, body }) => {
-                self.check_defaultness(item.span, *defaultness);
+            ItemKind::Fn(func) => {
```

Comment:
why did you change this?

Reply:
So I could use it in line 950. Let me change how I'm doing this.